tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "experimentalDecorators": true,
+    "checkJs": false,
```

Comment:
is this left over from when routeHelper was `.js`?

Reply:
Yeah that can go now.